### ISSUE TITLE
Cleanup Temporary UI CLI for connections and login pane

### ISSUE THREAD

**Issue by lane55** [pc2ccs/pc2v9#436]
**Is your feature request related to a problem?** Please provide a clear and concise description of what the problem is (for example: "I'm always frustrated when ... " )  

Temporary options/work should be removed after an appropriate testing time (around July 8th, one month test)

Remove these options, added in issue #432 

   private static final String NO_CONNECTIONS_PANE_OPTION_STRING = "--noconnectionspane";
   private static final String NO_LOGINS_PANE_OPTION_STRING = "--nologinspane";  

**Feature Description**: please provide a clear and concise description of the proposed feature:  

**Have you considered other ways to accomplish the same thing?** Please provide a clear and concise description of any alternative solutions or features you've considered.  
...

**Do you have any specific suggestions for how your feature would be ***implemented*** in PC^2?** If so, please describe.  
...

**Additional context**: please provide any additional information that will help us in considering your feature request.  
...

**Comment by clevengr** [comment]
I disagree that the options referenced in this issue are candidates for removal "after an appropriate testing time (around July 8th, one month test)".   These options provide the ability to keep the Connections and Logins panes from being displayed on the Admin GUI.  The options need to remain in place until the reason that the corresponding panes "hang" the Admin are found and fixed, not just until "some appropriate testing time" has elapsed.

I have no problem with the options being removed _after_ the reasons for needing them are found and fixed; I would have a large issue with removing them just because some time has passed.

Perhaps the real issue is that the title of this Issue should be something like "Connections and Logins panes result in inordinate delays with large contests", with an additional caveat that once the (underlying) issue is found and fixed, THEN the "temporary" options should be removed.

**Comment by lane55** [comment]
This is a proper issue as cleanup for the https://github.com/pc2ccs/pc2v9/issues/432 title "Temporary support for command-line suppression of Connections and Logins grids on Admin."  So either the change is Temporary or de facto permanent (as so many temporary changes have been in pc2.).   

The July 8th date has does as intended - to focus on this issue to fix/remove it much sooner than later/never.
